src/Carousel.tsx

```diff
@@ -46,9 +46,10 @@ const Carousel = React.forwardRef<ICarouselInstance, TCarouselProps<any>>(
 
         const commonVariables = useCommonVariables(props);
         const { size, handlerOffsetX } = commonVariables;
+        const dataLength = data.length;
 
         const offsetX = useDerivedValue(() => {
-            const totalSize = size * data.length;
```

Comment:
remove `data` dependence and add `dataLength`

Reply:
In order to ensure that there will be no new problems in reactive, it is recommended to keep `data` in `dependences`